Gruntfile.js

```diff
@@ -147,51 +147,52 @@ module.exports = function (grunt) {
   ])
 
   grunt.registerTask(
-    'test_default',
-    'Test that the default task runs the app',
-    [
-      'copy',
-      'sass'
-    ]
-  )
-
-  grunt.registerTask(
     'encode_snippets',
     'Encode HTML snippets',
     function () {
       grunt.task.run('htmlentities')
     }
   )
 
+  // Test
   grunt.registerTask(
```

Comment:
Might be my ignorance but do these need to be this verbose?

http://gruntjs.com/api/grunt.task

e.g.

```
task.registerTask('test', ['jshint', 'test_default', 'test_default_success']);
```